docs/src/main/sphinx/release/release-432.md

```diff
@@ -0,0 +1,57 @@
+# Release 432 (2 Nov 2023)
+
+:::{note}
+From this point forward, release notes will include a ⚠️ symbol to designate
+breaking changes. Breaking changes may:
+* Require users to update their configuration, such as if a property is removed
+* Update the default values for configuration properties, which may change
+  behavior
+* Require external systems or software to be updated alongside Trino, such as
+  when support for older dependencies is removed
+* Involve non-backwards compatible changes to the SPI which may require plugins
+  to be updated
+* Otherwise be deemed a significant change which requires specific attention
+  from teams managing a Trino instance
+:::
+
+## General
+
+* Improve performance of `CREATE TABLE AS ...` queries that contain a redundant
+  `ORDER BY` clause. ({issue}`19547`)
+* ⚠️ Remove support for late materialization, including the
+  `experimental.late-materialization.enabled` and
+  `experimental.work-processor-pipelines` configuration properties. ({issue}`19611`)
+* Fix query failure when using inline functions. ({issue}`19561`)
```

Comment:
in what scenario?

Reply:
Not entirely sure myself, the proposed release not just said "in some queries" and I'm not familiar enough with the code path to piece it together

Reply:
It's related to predicate pushdown, but the conditions under which it happens are too detailed to explain in a RN.

Reply:
I'll go with the old "potential" query failure.